scripts/windows-native-ci.ps1

```
@@ -972,8 +1018,25 @@ function Invoke-BuildArtifacts {
     $previousCgo = $env:CGO_ENABLED
     try {
         $env:CGO_ENABLED = '0'
-        Invoke-WindowsNativeProcess $go @('build', '-ldflags', "-s -w -X main.version=$packageVersion", '-o', (Join-Path $stage 'defenseclaw.exe'), './cmd/defenseclaw') -TimeoutSeconds 900 | Out-Null
-        Invoke-WindowsNativeProcess $go @('build', '-ldflags', "-s -w -H=windowsgui -X main.version=$packageVersion", '-o', (Join-Path $stage 'defenseclaw-hook.exe'), './cmd/defenseclaw-hook') -TimeoutSeconds 900 | Out-Null
+        $gateway = Join-Path $stage 'defenseclaw.exe'
+        $hook = Join-Path $stage 'defenseclaw-hook.exe'
+        Invoke-WindowsNativeProcess $go @('build', '-ldflags', "-s -w -X main.version=$packageVersion", '-o', $gateway, './cmd/defenseclaw') -TimeoutSeconds 900 | Out-Null
+        Invoke-WindowsNativeProcess $go @('build', '-ldflags', "-s -w -H=windowsgui -X main.version=$packageVersion", '-o', $hook, './cmd/defenseclaw-hook') -TimeoutSeconds 900 | Out-Null
+        Assert-WindowsExecutableResource $gateway 'gateway' $packageVersion -Apply
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Assert-WindowsExecutableResource' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)